src/components/BookForm.js

```diff
@@ -1,17 +1,48 @@
 import './BookForm.css';
+import { useDispatch, useSelector } from 'react-redux';
+import { useState } from 'react';
+import { addBook } from '../redux/books/books';
 
-const BookForm = () => (
-  <div className="book-form-wrapper">
-    <div>
-      <div>Add New Book</div>
-      <form className="book-form">
-        <input placeholder="Book title" className="book-title" />
-        <input placeholder="Book author" />
-        <button type="submit">Add Book</button>
-      </form>
-    </div>
+const BookForm = () => {
+  const dispatch = useDispatch();
+  const books = useSelector((state) => state.bookReducer);
+  const [title, setTitle] = useState('');
+  const [author, setAuthor] = useState('');
+
+  const addBookToStore = (e) => {
+    e.preventDefault();
+    const book = {
+      id: books.length,
+      title,
+      author,
+    };
+    dispatch(addBook(book));
+    setTitle('');
+    setAuthor('');
+  };
```

Comment:
- [ ] Optional suggestion

- You can validate the input so that the user couldn't add the empty data.